packages/components/addon/components/hds/form/sensitive-input/base.hbs

```diff
@@ -0,0 +1,33 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div class={{this.classNames}} {{style width=@width}}>
+  {{#if @isMultiline}}
+    <Hds::Form::Textarea::Base
+      class="hds-form-sensitive-input__control"
+      @value={{@value}}
+      @isInvalid={{@isInvalid}}
+      @height={{@height}}
```

Comment:
could it be that we have to forward the `@width` (for both of them)?

Reply:
The width would have to be set at the wrapper level (see L5) as the relative element, otherwise the icon may fall outside the input.

Reply:
Ahhh yes, of course! (I thought this was the "field" but it's not, it's the "base" of the sensitive input. Makes sense.